src/workspaces/WorkspacePolicies/WorkspacePolicies.tsx

```diff
@@ -0,0 +1,89 @@
+import { icon, useUniqueId } from '@terra-ui-packages/components';
+import _ from 'lodash/fp';
+import pluralize from 'pluralize';
+import React from 'react';
+import { ReactNode, useState } from 'react';
+import { BillingProject } from 'src/billing-core/models';
+import { Checkbox, LabeledCheckbox } from 'src/components/common';
+import colors from 'src/libs/colors';
+import * as Style from 'src/libs/style';
+import { getPolicyDescriptions, phiTrackingLabel, WorkspaceWrapper } from 'src/workspaces/utils';
+
+export type WorkspacePoliciesProps = {
+  workspace?: WorkspaceWrapper;
+  billingProject?: BillingProject;
+  title?: string;
+  policiesLabel?: string;
+  policiesLink?: ReactNode;
+  endingNotice?: ReactNode;
+  noCheckboxes?: boolean;
+  togglePhiTracking?: (selected: boolean) => void;
```

Comment:
Nit: conventionally, event callbacks like this are named `on...`.

```suggestion
  onTogglePhiTracking?: (selected: boolean) => void;
```

Reply:
Done